operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/ExportOperation.java

```diff
@@ -83,17 +83,15 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
         OperationConstants.ExportType exportType = export.checkExportType(operationContext.getType(), resourceType);
 
         if (!ExportType.INVALID.equals(exportType)) {
-            // For System $export, resource type(s) is required.
-            if (ExportType.SYSTEM.equals(exportType) && types == null) {
-                throw export.buildOperationException("Missing resource type(s)!", IssueType.INVALID);
-            }
-
             if (ExportType.PATIENT.equals(exportType)) {
                 if (types != null && !types.isEmpty()) {
                     export.checkExportPatientResourceTypes(types);
                 } else {
                     types = export.addDefaultsForPatientCompartment();
                 }
+            } else if ((ExportType.SYSTEM.equals(exportType) || ExportType.GROUP.equals(exportType))
```

Comment:
I was thinking that GROUP export should be handled more similar to PATIENT export.  It basically just the same except that the set of Patients to export is calculated from the group.

Reply:
Group does not have the compartment limitation

Reply:
Updated to Limit Export to Patient Compartment for $patient/$group export